artiq/coredevice/phaser.py

```diff
@@ -197,6 +200,11 @@ def init(self, debug=False):
             raise ValueError("large number of frame CRC errors")
         delay(.1*ms)  # slack
 
+        # determine the origin for frame-aligned timestamps
+        self.measure_frame_timestamp()
+        if self.frame_tstamp < 0:
+            raise RuntimeError("frame timestamp measurement timed out")
```

Comment:
```suggestion
            raise ValueError("frame timestamp measurement timed out")
```

to be consistent with the (admittedly suboptimal) choice in all the other `raises`.

Reply:
Changed in 4d7bd3e.